docs/faq.rst

```diff
@@ -165,6 +165,31 @@ Yes, if you want to detect accidental data damage (like bit rot), use the
 If you want to be able to detect malicious tampering also, use a encrypted
 repo. It will then be able to check using CRCs and HMACs.
 
+I am seeing 'A' (added) status for a unchanged file!?
+-----------------------------------------------------
+
+The files cache (which is used to determine whether |project_name| already
+"knows" / has backed up a file and if so, to skip the file from chunking)
+does intentionally *not* contain files that:
+
+- have >= 10 as "entry age" (|project_name| has not seen this file for a while)
+- have a mtime same as the newest mtime in the created archive
```

Comment:
s/mtime/modification time/

could we reword this? i am not sure what it means: "have been modified while the archive was created?" maybe?